src/test/java/com/browserstack/steps/OrderPageSteps.java

```diff
@@ -0,0 +1,28 @@
+package com.browserstack.steps;
+
+import io.cucumber.java.en.Then;
+import org.junit.jupiter.api.Assertions;
+import org.openqa.selenium.By;
+import org.openqa.selenium.NoSuchElementException;
+import org.openqa.selenium.WebElement;
+
+public class OrderPageSteps {
+
+    private final StepData stepData;
+
+    public OrderPageSteps(StepData stepData) {
+        this.stepData = stepData;
+    }
+
+    @Then("I should see elements in list")
+    public void iShouldSeeElementsInList() {
+        WebElement element = null;
+        try {
+            element = stepData.webDriver.findElement(By.cssSelector("#__next > main > div > div > h2"));
+            throw new AssertionError("There are no orders");
```

Comment:
Why direct throw AssertionError without checking any condition?

Reply:
if that h2 element is found then there are no orders.
So if there are orders then h2 element will not be available and hence element not found exception will be thrown.

Reply:
changed to positive case